services/resource-usage-tracker/tests/unit/with_dbs/test_api_licensed_items_checkouts.py

```diff
@@ -0,0 +1,139 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+# pylint:disable=too-many-arguments
+
+
+from datetime import UTC, datetime, timedelta
+from decimal import Decimal
+from typing import Generator
+
+import pytest
+import sqlalchemy as sa
+from models_library.api_schemas_resource_usage_tracker.licensed_items_checkouts import (
+    LicensedItemCheckoutGet,
+    LicensedItemsCheckoutsPage,
+)
+from models_library.resource_tracker_licensed_items_purchases import (
+    LicensedItemsPurchasesCreate,
+)
+from servicelib.rabbitmq import RabbitMQRPCClient
+from servicelib.rabbitmq.rpc_interfaces.resource_usage_tracker import (
+    licensed_items_checkouts,
+    licensed_items_purchases,
+)
+from simcore_postgres_database.models.resource_tracker_licensed_items_checkouts import (
+    resource_tracker_licensed_items_checkouts,
+)
+from simcore_postgres_database.models.resource_tracker_service_runs import (
+    resource_tracker_service_runs,
+)
+
+pytest_simcore_core_services_selection = [
+    "postgres",
+    "rabbit",
+]
+pytest_simcore_ops_services_selection = [
+    "adminer",
+]
+
+
+_USER_ID_1 = 1
```

Comment:
Use fixtures for these constants! 
You already have them in  `faker_users_data` and/or `faker_payments_data`. 
Promote the methods/fixtures that we already have.  If enhances maintainability since it makes it compatible with other existing tools/fixtures in place

Reply:
I will improve testing in upcoming PRs